### DISCUSSION TITLE
Learning rate schedule (example)

### DISCUSSION THREAD

**Discussion by RodrigoAVargasHdz** [jax-ml/jax#6192] [Q&A]
Hi, 
I was wondering if someone could show me an example of how to include any learning rate schedule in my optimization.

Thanks

**Comment by mattjj** [comment]
Thanks for the question!

This might be a good question for a [neural network library](https://github.com/google/jax#neural-network-libraries), like [Flax](https://github.com/google/flax), [Trax](https://github.com/google/trax), [Objax](https://github.com/google/objax), or [Haiku](https://github.com/deepmind/dm-haiku). You could try one of those issue trackers.

What do you think?

**Comment by dionhaefner** [comment]
FWIW, [optax supports schedules](https://github.com/deepmind/optax#schedules-schedulepy).
